extra-files/cloud_setup/tool_data_table_conf.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<!--
+    This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
+-->
+<tables>
+    <!-- Locations of all fasta files under genome directory -->
+    <table name="all_fasta" comment_char="#">
+        <columns>value, dbkey, name, path</columns>
+        <file path="/cvmfs/data.galaxyproject.org/byhand/location/all_fasta.loc" />
```

Comment:
No restrictions that I know of. On Friday, I was trying to build this same image as a container to make the whole setup much more portable but ran into issues w/ CVMFS (since it's using fuse) - expect an email from me expanding on this topic soon.